curriculum/challenges/english/10-coding-interview-prep/rosetta-code/comma-quibbling.md

```diff
@@ -52,19 +41,19 @@ assert(typeof quibble(['ABC']) === 'string');
 assert.equal(quibble(testCases[0]), results[0]);
 ```
 
-`quibble(["ABC"])` should return "{ABC}".
+`quibble(["freeCodeCamp"])` should return "{freeCodeCamp}".
 
 ```js
 assert.equal(quibble(testCases[1]), results[1]);
 ```
 
-`quibble(["ABC", "DEF"])` should return "{ABC and DEF}".
+`quibble(["freeCodeCamp", "Curriculum"])` should return "{freeCodeCamp and Curriculum}".
 
 ```js
 assert.equal(quibble(testCases[2]), results[2]);
 ```
 
-`quibble(["ABC", "DEF", "G", "H"])` should return "{ABC,DEF,G and H}".
+`quibble(["freeCodeCamp", "Curriculum", "Articles", "Volunteers"])` should return "{freeCodeCamp,Curriculum,Articles and Volunteers}".
```

Comment:
Do we really want this challenge to omit spaces? That's not consistent with the example above
```suggestion
`quibble(["freeCodeCamp", "Curriculum", "Articles", "Volunteers"])` should return "{freeCodeCamp, Curriculum, Articles and Volunteers}".
```